### PR TITLE
Curves related to Belyi maps

### PR DESCRIPTION
**Pull request by SamSchiavone**
I wrote scripts to check which Belyi maps are defined on curves in the LMFDB and, for those maps that are, added a link to the curve in the Related objects box on the Belyi map page.

I also had to modify names_and_urls in order to get pages for individual curves, not just isogeny classes.

### CODE REVIEW

**Comment by AndrewVSutherland** [comment]
pyflakes not happy with a python 2 print (and this print looks like a debug line that should be taken out in any case):

./scripts/belyi/friends/find_friends.py:49:39: invalid syntax
    print "Curve not found in database"

**Comment by SamSchiavone** [comment]
Thanks, Drew. I cleaned up the print statements and (hopefully) fixed all the other complaints pyflakes had.

**Comment by roed314** [comment]
There are still pyflakes failures.

You can run it locally by doing
```
$ sage -pip install pyflakes
$ sage -python -m pyflakes PATH_TO_LMFDB_ROOT
```

**Comment by SamSchiavone** [comment]
Ah sorry, I forgot about the missing imports. It should be fixed now.

File: scripts/belyi/find_friends.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+        E2 = EllipticCurve(ainvs2)
+        if E.is_isomorphic(E2):
+            return r['lmfdb_label']
+    #print("Curve not found in database")
+    return None
+
+def genus1_lookup_equation_nf(rec):
+    assert rec['g'] == 1
+    # make base field
+    nf_matches = list(db.nf_fields.search({'coeffs':rec['base_field']}))
```
Comment (roed314) [inline scripts/belyi/find_friends.py]: Is `coeffs` always in polredabs form?

Comment (SamSchiavone) [reply scripts/belyi/find_friends.py]: Yes, we always store the polredabs coefficients.

**Pull request merged**